arch/arm/core/aarch64/Kconfig

```diff
@@ -90,6 +90,95 @@ config GEN_ISR_TABLES
 config GEN_IRQ_VECTOR_TABLE
 	default n
 
+config ARM_MMU
+	bool "ARM MMU Support"
+	help
+	  Memory Management Unit support.
+
+if ARM_MMU
+
+choice
+	prompt "Page size"
```

Comment:
Do we need this if you are hardcoding `PAGE_SIZE` directly into the MMU code?

Reply:
yes right, I will just remove this.